### ISSUE TITLE
Backbone trigger change is not working

### ISSUE THREAD

**Issue by tonyfast** [bokeh/bokeh#2966]
All of the callbacks seem to be giving me the same errors after I try to trigger a change in the Backbone model.  The [Interact example](http://bokeh.pydata.org/en/latest/docs/user_guide/interaction.html#customjs-for-widgets) reproduces this problem.

The console error after ``.trigger('change')`` is 

```
Uncaught TypeError: Cannot read property 'previousAttributes' of undefined

n.extend._onModelEvent	@	callback.html:181
y	@	callback.html:181
g	@	callback.html:181
p	@	callback.html:181
c.trigger	@	callback.html:181
(anonymous function)	@	VM49:11
n.execute	@	callback.html:194
e.slide	@	callback.html:221
(anonymous function)	@	callback.html:221
(anonymous function)	@	callback.html:193
t.Widget._trigger	@	callback.html:219
t.widget._slide	@	callback.html:218
(anonymous function)	@	callback.html:219
t.widget._mouseCapture	@	callback.html:218
(anonymous function)	@	callback.html:219
t.widget._mouseDown	@	callback.html:217
(anonymous function)	@	callback.html:219
(anonymous function)	@	callback.html:217
K.event.dispatch	@	callback.html:189
o.dispatch	@	callback.html:223
m.handle	@	callback.html:188
```

Currently working on bokeh version ``0.10.0dev1-388-g240d65c``

**Comment by yellow-beard** [comment]
Pretty sure this is a duplicate of https://github.com/jashkenas/backbone/issues/3717

**Comment by Haraldson** [comment]
So when will 1.2.4 ship?

**Comment by bryevdv** [comment]
I don't see any console errors with that example currently (0.11.1)

**Comment by bryevdv** [comment]
In any event, this is an upstream issue, so marking it as such

**Comment by bryevdv** [comment]
That fix was merged and we now have version 1.3.3 of Backbone. I have not heard any other reports of problems, so I am closing this. Please re-open if there is a demonstrable problem that remains.
